src/components/sidebar.tsx

```diff
@@ -209,6 +214,28 @@ export function Sidebar({ className }: SidebarProps) {
             </Button>
           ))}
         </div>
+
+        <SidebarCategory>Site</SidebarCategory>
+        <div className="space-y-1">
+          {siteNavigation.map((item) => (
+            <Button
```

Comment:
This should just be a SidebarItem component, since it's used in four places that ideally stay in sync. I'd just chuck it at the top of this file and not export it to keep it all together.